docs/docs/reference/shards/index.md

```diff
@@ -0,0 +1,76 @@
+---
+authors: Fragcolor & contributors
+license: CC-BY-SA-4.0
+---
+
+# Shards
+
+This section contains a reference of all the shards available within the shards language.
```

Comment:
```suggestion
This section contains a reference of all the shards available within the Shards language.
```

Reply:
so we user "Shards" (with a capital) when referring to the language and "shards" when referring to the building blocks?